media/js/src/assetDetail/CreateSelection.jsx

```diff
@@ -11,8 +11,6 @@ export default class CreateSelection extends React.Component {
     render() {
         return (
             <React.Fragment>
-                <h3>Additional Selection Information</h3>
```

Comment:
We wanted to keep this header and remove the secondary one. "Add Selection Details"